projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-ai.php

```diff
@@ -45,6 +42,38 @@ public function __construct() {
 		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
 	}
 
+	/**
+	 * Checks if a given request is allowed to get AI data from WordPress.com.
+	 *
+	 * @return true|WP_Error True if the request has access, WP_Error object otherwise.
+	 */
+	public function permissions_check() {
+
+		if ( ! current_user_can( 'edit_posts' ) ) {
+			// Only post editors can access the endpoints
+			return new WP_Error(
+				'rest_forbidden',
+				__( 'Sorry, you are not allowed to access Jetpack AI help on this site.', 'jetpack' ),
+				array( 'status' => rest_authorization_required_code() )
+			);
+		}
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			// Only allow WPCOM simple sites and Atomic
+			if ( get_current_site()->id !== 1 ) {
```

Comment:
You might want to check the suggested implementation in Slack, in terms of allowing site level authentication (blog token) for AT site on WPCOM: p1673521187444489/1673452156.937829-slack-CBG1CP4EN